Marlin/src/pins/lpc1769/pins_MKS_SGEN_L_V2.h

```diff
@@ -0,0 +1,365 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2020 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+/**
+ * MKS SGen pin assignments
+ */
+
+#ifndef MCU_LPC1769
+  #error "Oops! Make sure you have the LPC1769 environment selected in your IDE."
+#endif
+
+#define BOARD_INFO_NAME   "MKS SGEN_L V2"
+#define BOARD_WEBSITE_URL "github.com/makerbase-mks"
+
+//
+// EEPROM, MKS SGEN_L V2.0 hardware has 4K EEPROM on the board
```

Comment:
Do we need to implement extra support for the onboard EEPROM? Is it the "AT24C32" referred to below?